packages/next-mdx/index.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (pluginOptions = {}) => (nextConfig = {}) => {
+  const extension = pluginOptions.extension || /\.mdx$/
+
+  return Object.assign({}, nextConfig, {
+    webpack (config, options) {
+      if (!options.defaultLoaders) {
```

Comment:
I think this if statement can be removed by now 🤔

Reply:
Yeah I think 3 major versions should be good to remove it 👍

Reply:
🙌